### PR TITLE
Add Key resource

### PR DESCRIPTION
**Pull request by flovouin**
Hi,

I needed to create an API key for one of my projects and I stumbled upon this Terraform provider. The `API key` resource didn't exist so I added it. I used it in my project and it works fine. However I'm no Terraform nor Go expert, so it might be worth proofreading it.

Would you mind reviewing this diff, and consider merging it to this provider if you think it's relevant?

Cheers,

### CODE REVIEW

**Comment by Preskton** [comment]
Hi, @flovouin - at first glance, this looks awesome! I will take a look this week and get things integrated in.

File: README.md [hunk 1]
```diff
@@ -43,6 +46,10 @@ resource "twilio_subaccount" "woomy" {
     friendly_name = "Woomy Subaccount #1"
 }
 
+resource "twilio_key" "woomy" {
```
Comment (Preskton) [inline README.md]: Do you mind updating to call the resource `twilio_api_key`?

**Review by Preskton — CHANGES_REQUESTED** [review]
Looks great - requesting one somewhat minor change -- renaming the resource to `twilio_api_key` for clarity.

**Review by Preskton — APPROVED** [review]
lgtm :tada:

**Comment by tedmiston** [comment]
I wonder how easy / difficult it would be to extend this to support API keys for subaccounts?

**Comment by childish-sambino** [comment]
@tedmiston We're launching the official provider soon to Pilot which includes this capability. See more at https://github.com/twilio/terraform-provider-twilio

**Pull request merged**